packages/block-library/src/navigation/index.php

```diff
@@ -185,11 +195,19 @@ function build_navigation_html( $block, $colors, $font_sizes, $level_zero = true
 				)
 			);
 		}
+
+		$html .= '</span>';
+
+		// Append submenu icon to top-level item
+		if ( $attributes['showSubmenuIcon'] && $level_zero && $has_submenu ) {
+			$html .= '<span class="wp-block-navigation-link__submenu-icon">' . render_submenu_icon() . '</span>';
```

Comment:
Is it an option to append the icon through CSS?

Reply:
Not at this moment - it's a fixed HTML (from `render_submenu_icon()`), but it sounds like a good idea to have it customizable. Probably better to implement in another PR, though.

Reply:
@frontdevde has been working on a similar implementation, adding [arrows indicator [wip]](https://github.com/WordPress/gutenberg/pull/19681/commits/91a7300f506bad8a3f42a78e7e14db86441b407c). We need to coordinate our job, here :-D.

Reply:
Also, not sure about adding a CSS class here. It can be tackled with the `has-child` one.

Reply:
Resolving due to https://github.com/WordPress/gutenberg/pull/19601/files#r368122076.